compose/startup-order.md

```diff
@@ -76,7 +76,27 @@ script:
     ```none
     command: ["./wait-for-postgres.sh", "db", "python", "app.py"]
     ```
+- You can also use no tool at all and include the check in the command.
 
+    ```
+        services:
+          practice_docker: 
+            image: dockerhubusername/practice_docker
+            ports: 
+              - 80:3000
+            command: bash -c 'while !</dev/tcp/db/5432; do sleep 1; done; python app.py'
```

Comment:
I like your proposal but I seems that `/dev/tcp` is not available in all distributions. I was thinking maybe

```
while !(nc -v -z server 4321); do echo "sleeping"; sleep 1; done; python app.py
```
But `nc/dns-utils` is also not available in a bunch of distros.

Do you have any other ideas?